org.eclipse.lsp4e/pom.xml

```diff
@@ -9,9 +9,13 @@
 	<packaging>eclipse-plugin</packaging>
 	<version>0.18.1-SNAPSHOT</version>
 
+	<properties>
+		<forceSkipCache>false</forceSkipCache>
+	</properties>
+
 	<build>
 		<plugins>
-			<plugin>
+			<!-- plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-clean-plugin</artifactId>
 				<version>3.3.2</version>
```

Comment:
Just delete if you think it's not worth keeping it.

Reply:
Yeah. It's just a draft yet.